### ISSUE TITLE
Support node 10

### ISSUE THREAD

**Issue by szmarczak** [zeit/pkg#415]
I've installed `node` using `nvm`.

```bash
$ node -v
v10.0.0

$ nvm ls
->      v10.0.0
default -> 10.0.0 (-> v10.0.0)
node -> stable (-> v10.0.0) (default)
stable -> 10.0 (-> v10.0.0) (default)
iojs -> N/A (default)
lts/* -> lts/carbon (-> N/A)
lts/argon -> v4.9.1 (-> N/A)
lts/boron -> v6.14.2 (-> N/A)
lts/carbon -> v8.11.1 (-> N/A)

$ pkg index.js
> pkg@4.3.1
> Targets not specified. Assuming:
  node10-linux-x64, node10-macos-x64, node10-win-x64
> Error! No available node version satisfies 'node10'
```

**Comment by chrismpettyjohn** [comment]
I am receiving same issue when trying to use PKG (install from npm)
```




Chriss-Mac-mini:backend chris$ pkg .
> pkg@4.3.1
> Error! No available node version satisfies 'node10'
Chriss-Mac-mini:backend chris$ clear
```

**Comment by presto9292** [comment]
I assume the problem is that no precompiled base binary exists for node 10

**Comment by shawwn** [comment]
I tried adding support for node 10.1.0 over at https://github.com/zeit/pkg-fetch/pull/36. If anyone could take a look, I'd appreciate it!

**Comment by ryandelvecchio** [comment]
Any update on when this will be completed?

**Comment by MrXyfir** [comment]
+1

**Comment by aurelhann** [comment]
It could be a great feature for pkg.... I'm nor sure it's here to post that comment... possible in pkg-fetch that contain v8 patches

**Comment by igorklopov** [comment]
Fixed in 4.3.2. Thanks to @shawwn!
